### ISSUE TITLE
Validation epoch loss is actually training epoch loss

### ISSUE THREAD

**Issue by sebpuetz** [stickeritis/sticker#29]
https://github.com/danieldk/sticker/blob/afd8c0b352aa98f7a96d1b380bb143020c2c11dc/sticker-utils/src/bin/sticker-train.rs#L128

should probably be

```Rust
let (loss, acc) = run_epoch(&tagger, &validation_tensors, false, lr);
``` 

or 

https://github.com/danieldk/sticker/blob/afd8c0b352aa98f7a96d1b380bb143020c2c11dc/sticker-utils/src/bin/sticker-train.rs#L137

should be changed to not print the validation loss, since what's printed now is actually the training loss from the current epoch.

**Comment by sebpuetz** [comment]
Fixed in 49dd401
